xeon_scripts/benchmarkMIC-build.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
-[ -e "$BIN_DATA_PATH" ] || BIN_DATA_PATH=/store/disk00/slava77/analysis/CMSSW_9_1_0_pre1-tkNtuple/run1000/2017/pass-4874f28/initialStep
-fin=${BIN_DATA_PATH}/PU70/10224.0_TTbar_13+TTbar_13TeV_TuneCUETP8M1_2017PU_GenSimFullINPUT+DigiFullPU_2017PU+RecoFullPU_2017PU+HARVESTFullPU_2017PU/memoryFile.fv3.clean.writeAll.recT.072617.bin
+[ -e "$BIN_DATA_PATH" ] || BIN_DATA_PATH=/data2/slava77/samples/2021/11834.0_TTbar_14TeV+2021/
+fin=${BIN_DATA_PATH}/AVE_70_BX01_25ns/memoryFile.fv5.default.210703-d239b45.bin
 
```

Comment:
this file has `SNB_` in the results prefix.
Do we even need it now ? (otoh, phi1 is still alive, so, perhaps OK)

Reply:
I am unsure about the current usage of this script, but as you notes phi1 is still there. So, I left it unchanged for the time being.

Reply:
Have added a note to the main description that this is not addressed in this PR, despite those scripts not being of common use anymore. Resolving this, since it'll be mentioned above for history.